components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -77,7 +77,7 @@ def update_compression_job_metadata(db_cursor, job_id, kv):
     field_set_expressions = [f"{k} = %s" for k in kv.keys()]
     query = f"""
         UPDATE {COMPRESSION_JOBS_TABLE_NAME}
-        SET {", ".join(field_set_expressions)}
+        SET {", ".join(field_set_expressions)}, update_time = NOW(3)
```

Comment:
Just curious, would `CURRENT_TIMESTAMP(3)` also work? If so, using `CURRENT_TIMESTAMP` can promote better consistency with the default value, though I agree the final inserted value might not be different.